### ISSUE TITLE
strip email from authors?

### ISSUE THREAD

**Issue by mtmorgan** [seandavi/BiocPkgTools#9]
Should `getPackageInfo()` strip email from authors? the output is submitted to `generateBiocPkgDOI()`. E.g., if I have

```
Authors: Martin Morgan <martin.morgan@RoswellPark.org>
```
Should the function return my name and email, or just my name?

**Comment by mtmorgan** [comment]
Additionally...
```
Authors: Iman Author and Iman Otherauthor
```
should be `Iman Author, Iman Otherauthor`, and stray whitespace before `,` needs to be removed

**Comment by seandavi** [comment]
We may need to look at the variety of cases to deal with. It would be great to begin to standardize what is in the DESCRIPTION file and then validate. Ideally, we could build such functionality into something like BiocCheck if code exists. The DOIs can be updated ad lib.

**Comment by seandavi** [comment]
This issue has been additionally addressed by #28. Will close for now, but the issue of author listing from DESCRIPTION files is still an open question.
